ES/ESmy004.xml

```diff
@@ -30,360 +30,665 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           <title type="complete" ref="LIT1461Gadlah#Introduction"/>
+                           <incipit xml:lang="gez">
+                        <locus target="#1ra"/>
+                         <hi rend="rubric">በስመ: አብ፡ ወወል</hi>ድ፡ ወመንፈስ፡ ቅዱ<hi rend="rubric">ስ፡ ዝንቱ፡ መጽሐ</hi>ፍ፡
+                         ሥራዓቶሙ፡ ለሐ<hi rend="rubric">ዋርያት፡ ባዕዳንሂ፡</hi>
+                         እ<supplied reason="lost" resp="MK">ም</supplied>፡ <supplied reason="lost">፸ወ፪</supplied>፡ <sic resp="MK">አርድእ፡</sic>
+                          <hi rend="rubric">ንኡሳን፡ ወዘኮነ፡ ኵ</hi>ሉ፡
+                         እምዜናሆሙ፡ ውስተ፡ ኵሉ፡
+                         ብሔር፡ ወተዝካረ፡ ስምዕ፡ እ<supplied reason="lost" resp="MK">ምኔ</supplied>ሆሙ፡
+                         ወእለ፡ አዕረፉ፡ እምኔሆሙ፡ ጸሎቶሙ፡ ወበረከቶሙ፡
+                         ትዕቀበነ፡ ለኵልነ፡ ሕዝበ፡ ክርስቲያን፡ ለዓለመ፡ ዓለም፡ አሜን፨
+                         <foreign xml:lang="en">
+                           <locus target="#1ra"/> In the Name of the
                            Father and the Son and the Holy Spirit. This (is) the book of the rule of
                            the Apostles and of others among 72 lesser Disciples, and of all from
                            their story that was in every country, and of the record (of those) of
                            them who (suffered) the martyrdom and of those of them who died. May
                            their prayer and their blessing take care of us, of all of us, Christian
-                           people. [Forever and] ever. Amen. ....  ]<locus target="#1va">1va</locus>, l.17] With thanks of the Lord and with the help of the Lord
-                           Jesus Christ, we begin [to write] the preaching of the blessed [saint]
-                           Peter, the chosen chief of the Apostles…</foreign>
-                     </incipit>
-                     <explicit>
-                        <locus target="#182va">182va</locus>, l.30 (Dionysius on the tribes of the
-                        Apostles): …ወአኰቴት፡ ለመንፈስ፡ ቅዱስ፡ ዘ[<locus target="#182vb">182vb</locus>]ወሀቦሙ፡
-                        ጥበበ፡ ወአእምሮ፡ ከመ፡ ይስ[n.l.] ውስተ፡ ኵሉ፡ አጽ[n.l.] ዓለም፡ በእንተ፡ ቅድስት፡ ሥላሴ፡ ፩[n.l.]ላክ፡
-                        ሎቱ፡ ስብሐት፡ ወክብር፡ ወዕበይ፡ እስከ፡ ለዓለመ፡ ዓለም፡ አሜን፨ ወአሜን፨ ለዘ፡ ጸሐፎ ወለዘ፡ አን[n.l.] ወለዘ፡
-                        ተርጐሞ፡ ለዘሰምዐ፡ ቃላቲሁ፡ በአሚን፡ ወበርትዕ፡ ሃይማኖት፡ ሕቡረ፡ [n.l.]ሐሮሙ፡ እግዚአብሔር፡ በመንግሥተ፡
-                        ሰማያት፡ ለዓለም፡ አሜን፨<foreign xml:lang="en">
-                           <locus target="#182va">182va</locus>, l.30 (Dionysius on the tribes of
-                           the Apostles): … And thanks-giving [be] to the Holy Spirit Who gave them
-                           knowledge and understanding so that [n.l.] in all [n.l.] the world
-                           concerning the Holy Trinity, one [God], to Whom [be] glory and honour and
+                           people. Forever and ever. Amen.
+                        </foreign>
+                            </incipit>
+
```

Comment:
nesting problem, closing /msItem tag is missing here

Reply:
(and there is then one too many at the end)

Reply:
ok